iotdb-core/datanode/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -1052,6 +1053,11 @@ public void loadProperties(Properties properties) throws BadNodeUrlException, IO
     loadIoTConsensusProps(properties);
   }
 
+  private void reloadIoTConsensusProps(Properties properties) {
+    loadIoTConsensusProps(properties);
+    DataRegionConsensusImpl.reloadConsensusConfig(conf);
```

Comment:
no need to pass conf，just call DataRegionConsensusImpl.reloadConsensusConfig() is ok? DataRegionConsensusImpl can use config to reproduce related confs